src/fastertransformer/th_op/multi_gpu_gptneox/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Please rename the folder to gptneox directly, don't add `multi_gpu`. 
GPT has such prefix because we have separate implementation for single gpu and multi gpu in the past.